src/seedu/addressbook/data/tag/UniqueTagList.java

```diff
@@ -94,18 +94,6 @@ public boolean contains(Tag toCheck) {
     }
 
     /**
-     * Adds a Tag to the list.
-     *
-     * @throws DuplicateTagException if the Tag to add is a duplicate of an existing Tag in the list.
-     */
-    public void add(Tag toAdd) throws DuplicateTagException {
```

Comment:
`AddressBook#addTag()` is the only method that uses `UniqueTagList#add()`, and similarly so with remove in [v1 2/3]. Of course, I can understand the arguments for having a separate issue + PR for unused methods in UniqueTagList.